src/util/script_database.hpp

```diff
@@ -0,0 +1,80 @@
+#include <unordered_map>
+#include <string>
+
+namespace big
+{
+    enum class ProtectionStatus
+    {
+        ALLOWED_NOTIFY,
+        BLOCK_ALWAYS,
+        BLOCK_IN_FREEMODE // Renamed from BLOCK_IN_FM
+    };
+
+    struct ScriptInfo
```

Comment:
The YimMenu code base uses `snake_case`, only  the R* classes and enums are an exception to this.

Reply:
Fixed with [ddb1e17](https://github.com/YimMenu/YimMenu/pull/3045/commits/ddb1e17a6c438c48d53fc190e7a36fa39999afb3).